content/node/trustlessStaking.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Trustless Third Party Staking
+menuTitle: Trustless Third Party Staking
+weight: 30
+aliases:
+  - /home/node/trustlessStaking
+description: Trustless Third Party Staking utilizes non-custodial feature to stake pokt without handing over custody of tokens to the Node Operator at any stage.
+---
+
+Trustless Third Party Staking utilizes non-custodial feature to stake pokt without handing over custody of tokens to the Node Operator at any stage. With this method, Node Operators have no ability to steal either the stake or the rewards. Clients can also unstake without any intervention of the node operator and get their funds back.
+
+**Before proceeding, please consult with your preffered Node Operator if they support this method of staking.**
+
+This document will show how to stake a node with 15k tokens and later upstake it to 60k.
+We stake with 15k first to minimize any loss resulting from mistake(eg. Typing incorrect output address or public key) 
+
+Non-Custodial Staking method involves the following addresses/wallets
+
+**Output Address**
+Never share the private key of your output address. Make ample backups of this!
+You should already have this, a wallet with your funds. This can also be referred to as the non-custodial address.
+
+
+**Operator Address**
+An Operator Address will be created by the Node Operator.  Node operator will manage(edit Domain/Chains) and run your node using the key of the operator address. 
+
+The very first stake initiated from the output address can establish a link between the operator address and the output address. Node Operator can then start running your node using the operator address. Any node rewards will be sent to the output address. Additionally; if unstaked, the unstaked funds will also go to the output address. An output address can be linked to multiple operator addresses
+
+## Introduction
+
+**Staking Bins**:
+You can stake 15k,30k,45k,60k tokens. A 30k pocket node will earn the same as two 15k nodes for instance. It is a good practice to stake 10 tokens on top of that.
```

Comment:
> You can stake 15k,30k,45k,60k

You can stake 15k, 30k, 45k, or 60k+ tokens per node. 

I think there should be a better definition of stake weighted rewards. Something like:

Stake-Weighted Servicer Rewards is a mechanism that offers higher rewards to node runners who stake more POKT on their nodes, with rewards determined by organizing nodes into bins based on their stake amount. The higher the stake, the greater the reward multiplier.

Before "A 30k pocket node will earn the same POKT rewards as two 15k nodes for instance."